lib/routes.js

```diff
@@ -20,62 +20,31 @@ var conferenceID = 0;
 
 module.exports = function(app){
 
-  // Process an inbound call from an inbound
-  // call made to one of the two numbers
+  // Process an inbound call from an inbound call made to one of the two numbers
   // we've set up
-  app.get('/answer_inbound', function(req, res) {
-    // increment the conference ID so
-    // that every call has a unique conference
-    conferenceID++;
+  app.get('/answer', function(req, res) {
+    var answer_url = 'http://'+process.env['DOMAIN']+'/on-answer?conference_id='+conferenceID
+    console.log(answer_url);
 
-    // create a new call from the number called
-    // to the call center
-    nexmo.calls.create({
```

Comment:
If you got rid of the nexmo code here, might as well take out the nexmo requre and instance at the top